### PR TITLE
Added appcenter-auth to scripts

### PR DESCRIPTION
**Pull request by jackycodes**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Updated scripts to account for appcenter-auth

Note: this PR depends on [590](https://github.com/microsoft/appcenter-sdk-react-native/pull/590)

### CODE REVIEW

File: scripts/bump-ios-sdk-version.sh [hunk 1]
```diff
@@ -34,13 +34,16 @@ fi
...
-# appcenter-push projects
+# appcenter-auth appcenter-push projects
 fileContent="$(cat ./appcenter-crashes/scripts/postlink.js)"
 echo "${fileContent/\'AppCenter\/Crashes\', version\: \'$oldiOSSdkVersion\'/'AppCenter/Crashes', version: '$newiOSSdkVersion'}" > ./appcenter-crashes/scripts/postlink.js
 
 fileContent="$(cat ./appcenter-analytics/scripts/postlink.js)"
 echo "${fileContent/\'AppCenter\/Analytics\', version\: \'$oldiOSSdkVersion\'/'AppCenter/Analytics', version: '$newiOSSdkVersion'}" > ./appcenter-analytics/scripts/postlink.js
 
+fileContent="$(cat ./appcenter-auth/scripts/postlink.js)"
+echo "${fileContent/\'AppCenter\/Analytics\', version\: \'$oldiOSSdkVersion\'/'AppCenter/Analytics', version: '$newiOSSdkVersion'}" > ./appcenter-auth/scripts/postlink.js
```
Comment (dhei) [inline scripts/bump-ios-sdk-version.sh]:
```suggestion
echo "${fileContent/\'AppCenter\/Auth\', version\: \'$oldiOSSdkVersion\'/'AppCenter/Auth', version: '$newiOSSdkVersion'}" > ./appcenter-auth/scripts/postlink.js
```

Comment (dhei) [inline scripts/bump-ios-sdk-version.sh]:
```suggestion
echo "${fileContent/\'AppCenter\/Auth\', version\: \'$oldiOSSdkVersion\'/'AppCenter/Auth', version: '$newiOSSdkVersion'}" > ./appcenter-auth/scripts/postlink.js
```

Comment (jackycodes) [reply scripts/bump-ios-sdk-version.sh]: Done

Comment (jackycodes) [reply scripts/bump-ios-sdk-version.sh]: Thanks for catching this

Comment (guperrot) [reply scripts/bump-ios-sdk-version.sh]: You don't need to modify locally for github suggestions, they can be applied directly (I already clicked apply).

**Pull request merged**